README.md

```diff
@@ -1,2 +1,2 @@
-# ci.common [![Build Status](https://github.com/OpenLiberty/ci.ant/workflows/CI/badge.svg)](https://github.com/OpenLiberty/ci.common/actions) [![Maven Central Latest](https://maven-badges.herokuapp.com/maven-central/io.openliberty.tools/ci.common/badge.svg)](http://search.maven.org/#search%7Cgav%7C1%7Cg%3A%22io.openliberty.tools%22%20AND%20a%3A%22ci.common%22) [![Build Status](https://travis-ci.com/OpenLiberty/ci.common.svg?branch=master)](https://travis-ci.com/OpenLiberty/ci.common) [![Build status](https://ci.appveyor.com/api/projects/status/dnxgfs6bxxmkfj69?svg=true)](https://ci.appveyor.com/project/wasdevb1/ci-common/branch/master) [![Codacy Badge](https://api.codacy.com/project/badge/Grade/3d7251d37b4d4b51bfa19d53314169c4)](https://www.codacy.com/app/wasdevb1/ci.common?utm_source=github.com&amp;utm_medium=referral&amp;utm_content=WASdev/ci.common&amp;utm_campaign=Badge_Grade)
-Common library for Open Liberty and WebSphere Liberty Maven and Gradle plugins. Used internally by [ci.gradle](https://github.com/wasdev/ci.gradle) and [ci.maven](https://github.com/wasdev/ci.maven).
+# ci.common [![Build Status](https://github.com/OpenLiberty/ci.common/workflows/CI/badge.svg)](https://github.com/OpenLiberty/ci.common/actions) [![Maven Central Latest](https://maven-badges.herokuapp.com/maven-central/io.openliberty.tools/ci.common/badge.svg)](http://search.maven.org/#search%7Cgav%7C1%7Cg%3A%22io.openliberty.tools%22%20AND%20a%3A%22ci.common%22) [![Codacy Badge](https://api.codacy.com/project/badge/Grade/3d7251d37b4d4b51bfa19d53314169c4)](https://www.codacy.com/app/wasdevb1/ci.common?utm_source=github.com&amp;utm_medium=referral&amp;utm_content=WASdev/ci.common&amp;utm_campaign=Badge_Grade)
```

Comment:
For the github actions badge, we can append `?branch=master` to have it only reflect the master branch build status.

Reply:
@kevinwlau Let me know if my last commit addressed your comment correctly.

Reply:
Looks good!